components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2521,6 +2529,110 @@ impl ToAnimatedZero for IntermediateSVGPaintKind {
     }
 }
 
+impl Into<NumberOrPercentage> for NonNegativeLengthOrPercentage {
```

Comment:
Thanks @nox.

I addressed it, But I think that we need those convert implementation for interpolating `SvgLengthOrPercentagOrNubmer` as `NumberOrPercentage`.
(e.g: interpolating between px<=>number)